apps/pwabuilder/src/script/components/manifest-info-card.ts

```diff
@@ -12,23 +12,26 @@ export class ManifestInfoCard extends LitElement {
   @property({ type: String }) field: string = "";
   @property({ type: String }) placement:  "" |"top" | "top-start" | "top-end" | "right" | "right-start" | "right-end" | "bottom" | "bottom-start" | "bottom-end" | "left" | "left-start" | "left-end" = "";
   @state() currentlyHovering: boolean = false;
+  @state() currentlyOpen: boolean = false;
+  @state() hoverTimer: any;
```

Comment:
could technically have the `number` type

Reply:
When I try to put it as a number it says 
Type 'Timeout' is not assignable to type 'number'. when I hover `this.hoverTimer` 
is this just a Typescript annoyance?

Reply:
> When I try to put it as a number it says 
> Type 'Timeout' is not assignable to type 'number'. when I hover `this.hoverTimer` 
> is this just a Typescript annoyance? 

Ahh that's weird, sounds like I'm wrong here then. Any is fine with me